stdlib/ballerina-builtin/src/main/ballerina/observe/tracing.bal

```diff
@@ -18,30 +18,80 @@ package ballerina.observe;
 
 import ballerina/http;
 
-@Description {value:"Reference types between spans"}
-@Field {value:"CHILDOF: The parent span depends on the child span in some capacity"}
-@Field {value:"FOLLOWSFROM: The parent span do not depend on the result of the child span"}
-@Field {value:"ROOT: Specify this as a root span that has no parent span"}
-public enum ReferenceType {
-    CHILDOF, FOLLOWSFROM, ROOT
-}
+public type ReferenceType "CHILDOF" | "FOLLOWSFROM" | "ROOT";
+
+@final public ReferenceType ReferenceType_CHILDOF = "CHILDOF";
```

Comment:
Shall we make these variables upper case..

Reply:
Fixed in commit 87a6fa0